lib/jsdom/living/structured-clone/structuredDeserializeWithTransfer.js

```diff
@@ -0,0 +1,448 @@
+"use strict";
+
+const DOMException = require("domexception/webidl2js-wrapper");
+
+function structuredDeserializeWithTransfer(serializeWithTransferResult, targetRealm) {
+  /**
+   * Let memory be an empty map.
+   * Let transferredValues be a new empty List.
+   */
```

Comment:
It's best not to copy the spec, since it gets out of date over time. Instead just provide a single-line comment linking to the relevant spec, so people can compare the spec and the code side-by-side.

Reply:
I copied it almost everywhere ahah.
I think we might keep them until the last moment. I mean, I do not expect specs to change in short time. Wdyt?